mantidproject/mantid
# Add lower bound to Delta function in Indirect data analysis

**Description of work.**
This PR adds a lower bound (0.0 < height) to the delta function defined in the ConvFit tab of the Indirect data analysis GUI.



**To test:**

1. Data to test the GUI can be found in #26631
2. Open the Indirect GUI
3. Go to the ConvFit tab
4. Add a delta function and confirm that the height is bounded below by 0.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
